src/Databases/Mongo/src/MongoDBAtlasVectorCollection.cs

```diff
@@ -0,0 +1,75 @@
+using LangChain.Databases.Mongo.Client;
+using MongoDB.Bson.Serialization;
+using MongoDB.Driver;
+
+namespace LangChain.Databases.Mongo;
+
+public class MongoDBAtlasVectorCollection(
+    IMongoContext mongoContext,
+    string indexName,
```

Comment:
```suggestion
```

Reply:
Unless i am missing something, we need this to hold the name of the index in Atlas.